src/compute-from-cog.ts

```diff
@@ -25,17 +38,14 @@ export const computeFromCog = async (cog: string, forceLegacyCompose: string) =>
   const version = getBalVersion(bal);
   logger.info(`District cog ${cog} is using BAL version ${version}`);
 
-  // Temporary check for testing purpose
-  // Check if cog is part of the accepted cog list
-  const isCogAccepted = acceptedCogList.includes(cog);
-
-  // Check if bal is using BanID on all address lines
+  // Check if bal is using BanID
   // If not, send process to ban-plateforme legacy API
+  // If the use of IDs is partial, throwing an error
   const useBanId = await checkIfBALUseBanId(bal, version);
 
-  if (!useBanId || !isCogAccepted) {
-    const message = `District cog ${cog} do not support BanID`;
-    logger.info(message);
+  if (!useBanId) {
+    logger.info(`District cog ${cog} is part of the whitelist but does not use BanID.`)
+    logger.info(`Sending BAL to legacy compose...`) 
```

Comment:
L'ancien message était probablement suffisant. (on précise déjà qu'il fait partie de la Whiteliste en ligne 29.)
Et, on peut, peut-être, réunir les deux `logger.info ?

```suggestion
    logger.info(`District cog ${cog} does not use BanID: Sending BAL to legacy compose...`) 
```